templates/spider.tmpl

```diff
@@ -21,22 +22,21 @@ class {{ classname }}(Spider):
 
             data = {
                 '_type': 'event',
-                'id': self._parse_id(item),
                 'name': self._parse_name(item),
                 'description': self._parse_description(item),
                 'classification': self._parse_classification(item),
-                'start_time': self._parse_start(item),
-                'end_time': self._parse_end(item),
-                'timezone': self._parse_timezone(item),
+                'start': self._parse_start(item),
+                'end': self._parse_end(item),
                 'status': self._parse_status(item),
```

Comment:
I think we'd also like to encourage people to use the _generate_status method. maybe move this below the _generate_id line and put `data['status'] = self._generate_status(data, text='')` ...